### DISCUSSION TITLE
Playwright webkit not supported by Fedora OS

### DISCUSSION THREAD

**Discussion by sandrokan15** [davidteather/TikTok-Api#982] [Q&A]
Hi,
I'm a Fedora 3.7 OS user, and on Fedora the Playwright webkit library is not supported, as by [comment on GitHub](https://github.com/microsoft/playwright/issues/14736).
Is there a way to use TikTok-Api, without webkit ?

thank you,
Ale

**Comment by davidteather** [comment]
Moved it over to using chrome if that helps. If not then you cannot use it :(
You could spin up docker container of ubuntu or something and might work
